packages/components/src/icons/CommitVoteSuccessIcon.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { colors } from "../styleConstants";
+
+export interface CommitVoteSuccessIconProps {
+  height?: number;
+  width?: number;
+}
+
+export const CommitVoteSuccessIcon: React.SFC<CommitVoteSuccessIconProps> = props => {
+  const width = (props.width || 100).toString();
+  const height = (props.height || 100).toString();
+
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width={width} height={height} viewBox={`0 0 ${width} ${height}`}>
```

Comment:
@am9u one more thing I just noticed the height and width in the viewBox should stay the original size. Changing this will change the size of the box but not the actual icon so it would either cut it off or have white space.

Reply:
Ah yeah, that's a good point. These probably won't need to be resized very often and the effort to make them resizeable may not be worth it right now. I'll make those changes.